python/python/glide/redis_client.py

```diff
@@ -228,6 +228,30 @@ async def execute_transaction(
         await response_future
         return response_future.result()
 
+    async def execute_script(
```

Comment:
execute_script => _execute_script

to communicate a private function in python

Reply:
please also fix execute_transaction => _execute_transaction